src/implementation/fs/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as H5P from '../../';
+
+import { IContentStorage } from '../..';
+
+import DirectoryTemporaryFileStorage from './DirectoryTemporaryFileStorage';
+import FileContentStorage from './FileContentStorage';
+import FileLibraryStorage from './FileLibraryStorage';
+import InMemoryStorage from './InMemoryStorage';
+
+export default function h5pfs(
+    editorConfig: H5P.IEditorConfig,
+    librariesPath: string,
+    temporaryStoragePath: string,
+    contentPath: string,
+    contentStorage?: IContentStorage
+): H5P.H5PEditor {
+    return new H5P.H5PEditor(
+        new InMemoryStorage(),
+        editorConfig,
+        new FileLibraryStorage(librariesPath),
+        contentStorage || new FileContentStorage(contentPath),
+        new H5P.TranslationService(H5P.englishStrings),
```

Comment:
I think the TranslationService should be constructed in H5PEditor using a default if no service is passed to the constructor. The fs implementation is not really the best place for it.